src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/CalculateSinglePayment.java

```diff
@@ -1,14 +1,80 @@
 package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
 
+import org.springframework.web.bind.MissingServletRequestParameterException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types.SinglePaymentResponse;
+
+
+/**
+ * 
+ * CalculateSinglePayment Controller.
+ * 
+ * @author 176899
+ *
+ */
 @RestController
 public class CalculateSinglePayment {
 
-  @RequestMapping("/singlepaymenturl")
-  public String singlepaymenturl(@RequestParam(value="name", defaultValue="defaultvalue") String name) {
```

Comment:
Is it forcing xml or we have to set the accept to `application/xml` we could do this at the service api

Reply:
The output type is dependant on the POM dependency:
<dependency>
     		<groupId>com.fasterxml.jackson.dataformat</groupId>
     		<artifactId>jackson-dataformat-xml</artifactId>
 </dependency>
by including this we automatically return XML format. If it's excluded, we get JSON. The content type should be set automatically too.